submissions/LGeorge007/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,83 @@
+ /* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/LGeorge007/a-tiny-JS-world
+
+   Web app: https://lgeorge007.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: "dog",
+   name: "Vasiliy",
+   legs: 4,
+   hands: 0,
+   gender: "male",
+   friends: ["Bublik", "Beem"],
+   saying: "Woof woof!"
+};
+
+const cat = {
+   species: "cat",
+   name: "Murkka",
+   legs: 4,
+   hands: 0,
+   gender: "female",
+   friends: ["Barsik", "Murzik", "Felix"],
+   saying: "Meow meow!"
+};
+
+const woman = {
+   species: "human",
+   name: "Larisa",
+   legs: 2,
+   hands: 2,
+   gender: "female",
+   friends: ["Irina", "Vlad", "Marina"],
+   saying: "Hi!"
+};
+
+const catWoman = {
+   species: "superhero human",
+   name: "Cat woman",
+   legs: 2,
+   hands: 2,
+   gender: "female",
+   friends: ["Batman", "all cats in the world"],
+   saying: cat.saying
+};
+
+const man = {
+   species: "human",
+   name: "Boris",
+   legs: 2,
+   hands: 2,
+   gender: "male",
+   friends: ["Vladimir", "Mariya", "Oksana"],
+   saying: "Hello!"
+};
+
+const message = function(inhabitant) {
+   
+   const {species, name, gender, hands, legs, friends, saying} = inhabitant;
+   
+   return [
+      `${saying}`,
```

Comment:
@LGeorge007 can you please explain why do you refer a variable in this particular way here?

Reply:
Yes. It really doesn't make sense. Why interpolate when there is only one value? When I was writing code at night, it seemed logical to me to bring everything to one form. Sorry. I 'll repare it.